CMakeLists.txt

```diff
@@ -25,7 +25,7 @@
 cmake_minimum_required(VERSION 3.12)
 
 project (PFUNIT
-  VERSION 4.11.0
+  VERSION 4.11.10
```

Comment:
```suggestion
  VERSION 4.11.1
```

Reply:
Odd typo - I know I did not intend "10"  OTOH the "0" key is nowhere near the "1" key.